### ISSUE TITLE
Netlify build fails due to plugin error

### ISSUE THREAD

**Issue by SathyaBhat** [cdeleeuwe/netlify-plugin-hugo-cache-resources#1]
I enabled this plugin and now it seems to be throwing this error

```
8:53:44 PM: ┌─────────────────────────────────────────────────────────────┐
8:53:44 PM: │ Plugin "netlify-plugin-hugo-cache-resources" internal error │
8:53:44 PM: └─────────────────────────────────────────────────────────────┘
8:53:44 PM: ​
8:53:44 PM:   Error message
8:53:44 PM:   Error: ENOENT: no such file or directory, open '/opt/build/cache/cwd/resources.netlify.cache.json'
8:53:44 PM: ​
8:53:44 PM:   Plugin details
8:53:44 PM:   Package:        netlify-plugin-hugo-cache-resources
8:53:44 PM:   Version:        0.1.4
8:53:44 PM:   Repository:     git+https://github.com/cdeleeuwe/netlify-plugin-hugo-cache-resources.git
8:53:44 PM:   npm link:       https://www.npmjs.com/package/netlify-plugin-hugo-cache-resources
8:53:44 PM:   Report issues:  https://github.com/cdeleeuwe/netlify-plugin-hugo-cache-resources/issues
8:53:44 PM: ​
8:53:44 PM:   Error location
8:53:44 PM:   In "onPostBuild" event in "netlify-plugin-hugo-cache-resources" from Netlify app
8:53:44 PM: ​
8:53:44 PM:   Error properties
8:53:44 PM:   {
8:53:44 PM:     errno: -2,
8:53:44 PM:     code: 'ENOENT',
8:53:44 PM:     syscall: 'open',
8:53:44 PM:     path: '/opt/build/cache/cwd/resources.netlify.cache.json'
8:53:44 PM:   }
8:53:44 PM: ​
8:53:44 PM:   Resolved config
8:53:44 PM:   build:
8:53:44 PM:     command: hugo
8:53:44 PM:     commandOrigin: ui
8:53:44 PM:     environment:
8:53:44 PM:       - REVIEW_ID
8:53:44 PM:       - HUGO_VERSION
8:53:44 PM:     publish: /opt/build/repo/public
8:53:44 PM:   plugins:
8:53:44 PM:     - inputs: {}
8:53:44 PM:       origin: ui
8:53:44 PM:       package: netlify-plugin-hugo-cache-resources
```

**Comment by cdeleeuwe** [comment]
@SathyaBhat Hmm, not entirely sure why it throws this error. I think it's related to https://github.com/netlify/build/issues/1411

**Comment by nushankodikara** [comment]
Same problem here! not sure what causing it

```js
Error: ENOENT: no such file or directory, open '/opt/build/cache/cwd/resources.netlify.cache.json'

In "onPostBuild" event in "netlify-plugin-hugo-cache-resources" from Netlify app

Error properties: { errno: -2, code: 'ENOENT', syscall: 'open', path: '/opt/build/cache/cwd/resources.netlify.cache.json' }
```

**Comment by cdeleeuwe** [comment]
Fixed in #2
